src/test/ui/impl-trait/multiple-lifetimes/error-handling.stderr

```diff
@@ -3,10 +3,16 @@ error: lifetime may not live long enough
    |
 LL | fn foo<'a, 'b, 'c>(x: &'static i32, mut y: &'a i32) -> E<'b, 'c> {
    |        -- lifetime `'a` defined here                   ^^^^^^^^^ opaque type requires that `'a` must outlive `'static`
+   |
+   = help: Consider replacing `'a` with `'static`
 help: to allow this `impl Trait` to capture borrowed data with lifetime `'a`, add `'a` as a constraint
    |
 LL | existential type E<'a, 'b>: Sized; + 'a
    |
 
+help: the following changes may resolve your lifetime errors
+   |
+   = help: replace `'a` with `'static`
```

Comment:
Re: first comment, this duplicates the information pointed out further up in this output.